sdk-ts/tests/vault.test.ts

```diff
@@ -0,0 +1,26 @@
+import * as anchor from "@project-serum/anchor";
+import { Connection, Keypair } from "@solana/web3.js";
+import { expect } from "chai";
+import Vault from "../src/vault";
+import { SOL_MINT } from "../src/constants";
+
+const mockWallet = new anchor.Wallet(Keypair.generate());
+const mockConnection = new Connection("https://api.mainnet-beta.solana.com");
+
+test("Vault State", async () => {
+  const vault = new Vault(mockWallet, mockConnection);
+  await vault.getVaultStateByMint(SOL_MINT);
+
+  const lpSupply = await (await mockConnection.getTokenSupply(vault.state.lpMint)).value.amount;
```

Comment:
I think `(await mockConnection.getTokenSupply(vault.state.lpMint)).value.amount` should be sufficient.